tests/integration/aps/test_utils.py

```diff
@@ -23,49 +23,56 @@ def dag():
     return dagbag.get_dag(dag_id=DAG_NAME)
 
 
-def test_dag_loaded(dag: DAG):
-    assert dag is not None
-    assert len(dag.tasks) == 3
-
+@pytest.fixture
+def dag_was_paused(dag):
```

Comment:
again, better move it into one file and reuse it in other tests

Reply:
I need to use it as a fixture because I need to preserve the initial state, since I am unpausing during the test (if the dag was paused) and then returning the dag for initial state as it was before the running the tests